observability/monitoring/reviewing-monitoring-dashboards.adoc

```diff
@@ -6,40 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-ifndef::openshift-dedicated,openshift-rosa[]
-{product-title} {product-version} provides a comprehensive set of monitoring dashboards that help you understand the state of cluster components and user-defined workloads.
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
-{product-title} provides monitoring dashboards that help you understand the state of user-defined projects.
```

Comment:
@juzhao This division did not make sense to me because in the admin section we say that `dedicated-admin` can check core platform components. So I just left the first line for both OCP and ROSA/OSD. Let me know if that is wrong. Thank you!

Reply:
@eromanova97 you're right